### ISSUE TITLE
vite example

### ISSUE THREAD

**Issue by michelmar** [epam/ketcher#1787]
I'm having some hard time to configure ketcher to run on vite, I have tried in the standalone and in the remote server and I get the same error:
```javascript
TypeError: undefined is not an object (evaluating 'ketcher.eventBus')
Error: {}
```
The Editor is loaded but I can't use any ketcher funcionality, for example:
`TypeError: null is not an object (evaluating 'editor3.tool')
Error: {}`

I have checked that the resources are being returned from the indigo service.

In the funtion onInit the ketcher objetc is returned just as `{remote:true}` or `{standalone:true}`

Could someone help me?

**Comment by TatianaGrigorovich** [comment]
@michelmar, hello!
Which version of React are you using? This error may occur if you use React18.
I'm going to update Ketcher to React18 in [this issue](https://github.com/epam/ketcher/issues/1681)

**Comment by michelmar** [comment]
@TatianaGrigorovich Yeah that's right.  I'm using React 18. Thank you in advance.  Hope to see it working 🙏
